lib/event-handler.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+const {getProperty, hasProperty, setProperty} = require('./utils.js')
+
+module.exports = {
+  parseEvent
+, processEvent
+}
+
+function parseEvent(event) {
+  if (!hasProperty(event, 'Records')) return undefined
+
+  const records = getProperty(event, 'Records')
+  const checkRecords = Array.isArray(records) && records.length > 0
+  if (!checkRecords) return undefined
```

Comment:
minor, but you really only need `getProperty` and `hasProperty` when you're dealing with nested props. same is true throughout.

also simpler suggestion:

```js
const records = event.Records
if (!Array.isArray(records) || !records.length) return
```